controllers/benches/finder.js

```diff
@@ -0,0 +1,66 @@
+const { Op } = require('sequelize');
+const { Warehouse, Sticker, team } = require('../../databases/db');
+const responses = require('../../utils/responses/responses');
+
+
+/**
+ * Find receives a page number and a page size. It gets the user's paginated bench from their warehouse 
+ * and returns the paginated result.
+ * @param req - the request object
+ * @param res - the response object
+ */
+const find = async (req, res) => {
+  try {
+    let { page = 0, size = 10, playername: playerName = '.*', teamname = '%', position = ['arquero', 'defensa', 'delantero', 'medio campo'] } = req.query;
```

Comment:
Con razon se andaban cayendo a coñazos ayer, los IDENTIFICADORES del backend son eso, IDENTIFICADORES no deben (necesariamente) corresponder al texto que se mostrara en el frontend, esto va en INGLES

Incluso lo dice la documentacion: https://www.notion.so/Gestion-de-cromos-961d780d45b942bdbb802d02de0687b4

```suggestion
    let { page = 0, size = 10, playername: playerName = '.*', teamname = '%', position = ['goalkeeper', 'defender', 'forward', 'midfield'] } = req.query;
```

Reply:
Chito

Reply:
Aparte, el peo es que en la db de producción y staging, medio campo está guardado como MedioCampo, 'tonces igual esto doesn't works

Reply:
Explicate